backend/Testing/Browser/Base/PageTest.cs

```diff
@@ -39,6 +59,19 @@ await Context.Tracing.StartAsync(new()
                 Sources = true
             });
         }
+
+        Context.Response += (_, response) =>
+        {
+            if (response.Status >= (int)HttpStatusCode.InternalServerError)
+            {
+                DeferredExceptions.Add(new UnexpectedResponseException(response));
+            }
+        };
+
+        Context.Request += (_, request) =>
+        {
+            Console.WriteLine(request.IsNavigationRequest);
```

Comment:
I don't think this will show up properly in an xunit test. Was this left in by accident?

Reply:
Yes, I didn't intend to check that in. I'll remove it.